src/Symfony/Component/Intl/Tests/LanguagesTest.php

```diff
@@ -954,19 +1548,19 @@ public function testGetAlpha3Name($displayLocale)
     {
         $names = Languages::getNames($displayLocale);
 
-        foreach ($names as $alpha2 => $name) {
-            $alpha3 = self::$alpha2ToAlpha3[$alpha2] ?? false;
-            if ($alpha3) {
-                $this->assertSame($name, Languages::getAlpha3Name($alpha3, $displayLocale));
+        foreach ($names as $alpha2Code => $name) {
+            $alpha3Code = self::$alpha2ToAlpha3[$alpha2Code] ?? (3 === strlen($alpha2Code) ? $alpha2Code : null);
+            if (null !== $alpha3Code) {
+                $this->assertSame($name, Languages::getAlpha3Name($alpha3Code, $displayLocale));
             }
```

Comment:
Here I would like to have an else clause like this
```
else {
    $this->assertSame($name, Languages::getAlpha3Name($alpha2Code, $displayLocale));
}
```

Reply:
improved test similar like `getName`